README.md

```diff
@@ -78,7 +78,11 @@ Seed the database with the test users (refer to the source code for the user det
 
 ## Running the Development Server
 
-To start up the development server with live reloading of python code changes (`r` flag) and debug output (`d` flag):
+Begin by running the celery daemon:
+
+    $ celery -A _15thnight.celery worker
+
+and to start up the development server with live reloading of python code changes (`r` flag) and debug output (`d` flag):
```

Comment:
Change this text to:

    Then start up the development web server with live reloading of python code changes (`r` flag) and debug output (`d` flag):